theano/sandbox/cuda/tests/test_dnn.py

```diff
@@ -489,9 +489,15 @@ def test_dnn_conv_merge():
 
```

Comment:
line 484 have a semantic error. The shape should be (b, f, ih-iw+1, iw-kw+1). Can you change the value of ih to be 5 to make this detected?